documentation/dsls/DSL-AshEvents.EventLog.md

```diff
@@ -31,7 +31,7 @@ end
 
 | Name | Type | Default | Docs |
 |------|------|---------|------|
-| [`primary_key_type`](#event_log-primary_key_type){: #event_log-primary_key_type } | `:integer \| Ash.Type.UUIDv7` | `:integer` | The type of the primary key used by the event log resource. Valid options are :integer, :uuid_v4, and :uuid_v7. Defaults to :integer. |
+| [`primary_key_type`](#event_log-primary_key_type){: #event_log-primary_key_type } | `:integer \| Ash.Type.UUIDv7` | `:integer` | The type of the primary key used by the event log resource. Valid options are :integer  and :uuid_v7. Defaults to :integer. |
```

Comment:
Should it default to integer? If `:uuid_v7` "just works" for more cases, and provides strict ordering of events?